src/clj/fluree/db/api/query.cljc

```diff
@@ -215,36 +215,36 @@
         (dataset db-map defaults)))))
 
 (defn query-connection-fql
-  [conn query]
+  [conn query {:keys [role did]}]
   (go-try
-    (let [{query :subject, did :did} (or (<? (cred/verify query))
-                                         {:subject query})
-          {:keys [t opts] :as query*}  (update query :opts sanitize-query-options did)
+    (let [credentialed-opts-query     (update query :opts #(merge {:role role :did did} %))
+          {query :subject, did :did}  (or (<? (cred/verify credentialed-opts-query))
```

Comment:
I don't think we can modify the query and still expect to verify the results here. The query may not even be a map, it could be just a JWS string.

Reply:
If the query is just a JWS string, would there be some way to add the `role` and `did` to it?

Reply:
`cred/verify` returns a map of `:subject` and `:did`, if whatever it is verifying is valid. I don't think we should override the `did`, though - the supplied did should only be used if a query (or txn) is not signed.

Reply:
Ok, it sounds like I should just be inserting the :role and :did when we convert the sparql to fql, that way it won't mess with any JWS style queries that are submitted.

Reply:
Although, this is part of the `query-connection-fql` function, so can that be a JWS string? It sounds like it would have to be fql.